pcap.go

```diff
@@ -360,6 +360,7 @@ func (p *Pcap) getNextPkt(pkt *Packet) {
 		pkt.Caplen = C.MAX_PKT_CAPLEN
 	}
 	pkt.Data = (*[C.MAX_PKT_CAPLEN]byte)(buf)[0:pkt.Caplen]
+	pkt.Payload = pkt.Data
```

Comment:
Does data collector use this Payload?

Reply:
it does, in netflow/pcap.go:
26 func (p *pcapPkt) pktPayload() []byte {
27     return p.pkt.Payload
28 }

However, before my changes - the p.Payload get updated almost immediately in  pcap.Decode():

p.Payload = p.Data[14:]

Since we call pcap.Decode() first thing in capture.go, after we receive the packet - we should be ok.